cql2pgjson/src/test/java/org/folio/cql2pgjson/CQL2PgJSONTest.java

```diff
@@ -129,8 +128,13 @@ public void select(CQL2PgJSON aCql2pgJson, String sqlFile, String cql, String ex
       assertEquals("CQL: " + cql + ", SQL: " + sql, expectedNames, actualNames);
     } catch (QueryValidationException | SQLException e) {
       logger.fine("select: " + e.getClass().getSimpleName()
-        + " for query " + cql + " : " + e.getMessage());
+          + " for query " + cql + " : " + e.getMessage());
       if (! expectedNames.startsWith("!")) {
+        int pos = e.getMessage().indexOf("Position:");
+        if (pos != -1) {
+          int errorPos = Integer.parseInt(e.getMessage().substring(pos + 10)) - 1;
+          sql = sql.substring(0, errorPos) + "****^****" + sql.substring(errorPos);
```

Comment:
These two lines should be wrapped into
```
try {
  ....
} catch (Exception e) {
  // ignore
}
```
because parseInt may fail if there is more text after the position number.